README.md

```diff
@@ -1 +1,136 @@
 
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
Well done on the README 👍🏽 but as requested by the previous reviewer [here](https://github.com/OTMANE-ECHCHAFYKY/hello-microverse/pull/1) all the comments should be removed ❌  and these are considered comments as well. 🤔